charts/px-backup/templates/NOTES.txt

```diff
@@ -6,7 +6,7 @@ Monitor PX-Backup Install:
 --------------------------------------------------
 Wait for px-backup status to be in "Completed" state.
 
-    kubectl get po --namespace {{ .Release.Namespace }} -ljob-name=pxcentral-post-install-hook  -o wide | awk '{print $1, $3}' | grep -iv error
+    kubectl --namespace {{ .Release.Namespace }} wait --for=condition=complete --timeout=120s job/pxcentral-post-install-hook
```

Comment:
Increase the timeout to 5 mins, pulling images can take much longer than 2 mins

Reply:
changed.